ipynb/primary-processing/11-generate_per_bodysite_results.md

```diff
@@ -0,0 +1,253 @@
+Analysis focused on a paricular dataset often starts with seperating the data by bodysite. This notebook, and subsequent notebooks (12-15) will seperate the American Gut dataset by bodysite and rarefaction depth.
+
+```python
+import itertools
+import os
+
+import numpy as np
+import pandas as pd
+
+import americangut.util as agu
+import americangut.notebook_environment as agenv
+
+chp_path = agenv.activate('11-packaged')
+```
+
+We're also going to set up a series of reference variables, which we'll use through this analysis.
+
+```python
+alpha_metrics = {'pd': 'PD_whole_tree',
+                 'chao1': 'chao1',
+                 'shannon': 'shannon',
+                 'observedotus': 'observed_otus'
+                }
+depths = agenv.get_rarefaction_depth()
+depth_names = ['1k', '10k']
+```
+
+We'll start by splitting the raw OTU table and metadata by the `BODY_HABITAT` field, so we have an OTU table for each bodysite.
+
+```python
+for trim in ['notrim', '100nt']:
+    for depth in ['raw', '1k', '10k']:
+        meta = agu.get_existing_path(agenv.paths['package']['split']['%s-map' % trim])
+        otus = agu.get_existing_path(agenv.paths['package']['split']['%s-%s-otu' % (trim, depth)])
+        odir = agu.get_new_path(agenv.paths['package']['split']['%s-%s-dir' % (trim, depth)])
+
+        !mkdir -p $odir
+        !split_otu_table.py -i $otus \
+                            -m $meta \
+                            -o $odir \
+                            -f BODY_HABITAT
+```
+
+Next, we'll move the unrarefied tables, split by bodysite, for each of the bodysites we're interested in. For this, we're going to write a quick parameter iterator, that will let us generate the samples we need.
```

Comment:
Because I want them named in a specific way that doesn't match what's used in `split_otu_table.py`.

I also like to keep the steps distinct, because I like to be able to go back a step if I make a mistake with the metadata and save something I shouldn't when I'm doing analysis.